src/backend/distributed/commands/cascade_table_operation_for_connected_relations.c

```diff
@@ -116,6 +143,43 @@ LockRelationsWithLockMode(List *relationIdList, LOCKMODE lockMode)
 }
 
 
+/*
+ * HasPartitionRelationInvolvedInAnyNonInheritedForeignKeys is a wrapper function that
+ * returns true if FindPartitionRelationInvolvedInAnyNonInheritedForeignKeys can find
+ * a matching relation.
+ */
+static bool
+HasPartitionRelationInvolvedInAnyNonInheritedForeignKeys(List *relationIdList)
```

Comment:
thinking about better names for `HasPartitionRelationInvolvedInAnyNonInheritedForeignKeys`, `FindPartitionRelationInvolvedInAnyNonInheritedForeignKeys` & `RelationInvolvedInAnyNonInheritedForeignKeys`

Reply:
removed those functions, but finding a better name still applies to `ErrorIfHasPartitionRelationInvolvedInNonInheritedForeignKey`